oak_marketplace_emotion_recognition/README.md

```diff
@@ -0,0 +1,42 @@
+This contains the code for \***\*. For more information - visit [\*\***]()
+
```

Comment:
# Emotion Recognition 

Just after that title here should be a gif with new visualization and a pie chart.